src/language-js/embed.js

```diff
@@ -616,8 +611,27 @@ function printHtmlTemplateLiteral(
     }
   );
 
+  const firstWhitespace = node.quasis[0].value.raw.match(/^\s/) ? " " : "";
+  const lastWhitespace = node.quasis[node.quasis.length - 1].value.raw.match(
+    /\s$/
+  )
+    ? " "
+    : "";
+
+  if (firstWhitespace && lastWhitespace) {
+    return group(
+      concat([
+        "`",
+        indent(concat([ifBreak("", " "), softline, group(contentDoc)])),
```

Comment:
I guess you can use `line` instead of `ifBreak("", " "), softline`.
See https://github.com/prettier/prettier/blob/master/commands.md#line

Also you can probably assign `line` to `firstWhitespace` and `lastWhitespace` instead of `" "`.